tslearn/clustering/kmeans.py

```diff
@@ -610,12 +605,12 @@ def _fit_one_init(self, X, x_squared_norms, rs):
             self.cluster_centers_ = self.init.copy()
         elif self.init == "k-means++":
             if self.metric == "euclidean":
-                self.cluster_centers_ = _k_init(
+                self.cluster_centers_ = _kmeans_plusplus(
                     X.reshape((n_ts, -1)),
                     self.n_clusters,
                     x_squared_norms,
```

Comment:
To make it works, you also need to identify last 2 parameters :
x_squared_norms = x_squared_norms[0] (because of the dimension needed here)
random_state = rs

_kmeans_pluplus take only 2 positional arguments

Reply:
@ad299 I will properly specify the last 2 input arguments to the `_kmeans_plusplus` function. However, I'm not sure that only the first row of `x_squared_norms` needs to be specified. The `_k_init` function from `scikit-learn` 0.23 and this `_kmeans_plusplus` function from 0.24 take the `x_squared_norms` input argument with identical dimensions.

Reply:
@ad299 Could you explain why the `x_squared_norms` argument should only be passed the the first row of the`x_squared_norms` variable created in this file? I ask because `_kmeans_plusplus` from `scikit-learn` 0.24 and `_k_init` from 0.23 accept the `x_squared_norms` argument with the same dimensions.

Reply:
When i test it without selecting the first row I have an error, this is why. But if both functions need the same dimensions dont consider my test, problem is probably on an other part  in my test !